src/get-transaction-path.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = (transaction) ->
+  origin = transaction['origin']
+  delimiter = ":"
```

Comment:
Good point, I forgot to add escaping with a backslash mentioned here: https://github.com/apiaryio/dredd/issues/227